js/src/constants.js

```diff
@@ -141,3 +141,6 @@ export const PREFERENCES_STORE_NAMESPACE =
 
 export const PMAX_IMPROVE_PERFORMANCE_MAX_AD_STRENGTH =
 	'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH';
+
+export const PMAX_ASSETS_IMPROVEMENTS_BANNER_CONTEXT =
```

Comment:
@ankitrox Since we are using this constant only in `js/src/components/pmax-improve-assets-banner/banner.js`, let's move it there instead of `js/src/constants.js`